mobileapp/src/commonMain/kotlin/io/github/landrynorris/multifactor/integration/apps/App.kt

```diff
@@ -0,0 +1,3 @@
+package io.github.landrynorris.multifactor.integration.apps
+
+open class App(val name: String, val iconUrl: String, val packageName: String)
```

Comment:
This is currently defined in shared code. On iOS, we cannot currently make a custom autofill service, so recognizing the app's package name should only be relevant on Android. I figure there's no harm to tracking Android package name of other apps on iOS, but I don't know if we need to track the Bundle Identifier for iOS.

Reply:
This code will likely move to its own module later so autofill can access it regardless, so we may make that module Android only.